src/IECoreHoudini/plugin/Plugin.cpp

```diff
@@ -98,36 +98,45 @@ void newSopOperator(OP_OperatorTable *table)
 {
 	OP_Operator *opHolder = new OP_Operator(
 		"ieOpHolder", "Cortex Op",
-		SOP_OpHolder::create, SOP_ParameterisedHolder::parameters, 0, 4,
-		SOP_ParameterisedHolder::variables, OP_FLAG_GENERATOR
+		SOP_OpHolder::create,
```

Comment:
your formatting is nicer, but lets just go back to the original formatting to minimize the diff